crates/build/re_build_info/src/build_info.rs

```diff
@@ -147,6 +155,7 @@ impl CrateVersion {
 fn crate_version_from_build_info_string() {
     let build_info = BuildInfo {
         crate_name: "re_build_info",
+        features: "default extra",
```

Comment:
what's the `extra` about?

Reply:
Just a placeholder for some feature name - this is just for a test